src/test/java/Homework16.java

```diff
@@ -0,0 +1,31 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest{
+
+@Test
+public static void Navigation() {
+
+
+    WebDriver driver = new ChromeDriver();
+    driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+    String url = "http://bbb.testpro.io/";
+    driver.get(url);
+
+    WebElement registration = driver.findElement(By.cssSelector("a[id='hel']"));
+    registration.click();
+
+    WebElement userAvatar = driver.findElement(By.xpath(("/html/body/div/hs")));
```

Comment:
if you are beginner does not know who to write the xpath then you can relay on the xpath that will be provided by chrome browser. Right click on element that you are interested in then click on Inspect. Now right click on the html code of the element Copy > Copy Xpath. Now you have the xpath in the clipboard for you.

For example, here is the locator (Copy > Copy Xpath):  //*[@id="sidebar"]/section[1]/ul/li[3]/a

BUT I recommend not using plugins or copying the xpath from the page code using a browser or web developer.

For example, here is the correct locator:  //a[@class='songs']

You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/209771467-5c42031c-34d7-4ec4-8c96-154f23a59f31.png)